Makefile

```diff
@@ -57,12 +59,29 @@ pull: .env
 
 rebuild: clean build
 
+# Run Bedrock in default mode, serving all of Mozorg and also Pocket as /externalpages/pocket/*
+# TODO: amend this when the Pocket-mode split has been completed to ask if the user
+# meant run-pocket[-prod] or run-mozorg[-prod]
 run: .docker-build-pull
 	${DC} up assets app
 
 run-prod: .docker-build-pull
 	${DC} up release-local
 
+# Run in Mozorg-only mode, using Bedrock to serve ONLY Mozorg pages
+run-mozorg: .docker-build-pull
+	-SITE_MODE=${MOZORG_MODE} ${DC} up assets app
+
+run-mozorg-prod: .docker-build-pull
+	-SITE_MODE=${MOZORG_MODE} ${DC} up release-local
+
+# Run in Pocket-only mode, using Bedrock to serve ONLY Pocket pages _at the root path_
+run-pocket: .docker-build-pull
+	-SITE_MODE=${POCKET_MODE} ${DC} up assets app
+
+run-pocket-prod: .docker-build-pull
+	-SITE_MODE=${POCKET_MODE} ${DC} up release-local
+
```

Comment:
I would rename these to be similar to `npm run in-pocket-mode` etc, but when one factors in the prod option as well, it gets a tiny bit over-wordy (`make run in-pocket-prod-mode`). Opinions welcome over consistency vs UX

Reply:
make doesn't make it easy to pass in args to a command. What you have above would try to run both a `run` target and an `in-pocket-prod-mode` one.

Reply:
Fair point. `make run-in-pocket-mode` (two strings, thereby dodging the issue) would also be imperfect in a different way, as it might cause confusion when the FE team are used to `npm run in-pocket-mode` (three strings)

Will leave it as-is currently and we can get on with our lives 😆

Reply:
Though I think if we moved to `just` that it would allow for things like this.

Reply:
For now +1 on keeping as is and we'll evaluate later if/when we move away from `make`.